wpilibj/src/main/java/edu/wpi/first/wpilibj/ADXL345_I2C.java

```diff
@@ -231,15 +232,18 @@ public AllAxes getAccelerations() {
   @Override
   public void initSendable(NTSendableBuilder builder) {
     builder.setSmartDashboardType("3AxisAccelerometer");
-    NetworkTableEntry entryX = builder.getEntry("X");
-    NetworkTableEntry entryY = builder.getEntry("Y");
-    NetworkTableEntry entryZ = builder.getEntry("Z");
+    DoublePublisher pubX = new DoubleTopic(builder.getTopic("X")).publish();
```

Comment:
ADXL345 is an accelerometer, not a gyro.  And the C++ implementation is different because it's C++ (e.g. the lambda captures in C++ own the publishers).